### PR TITLE
Add /code-of-conduct

### LINKED ISSUE #539: 'Community and Support': Add a code of conduct
From URL: https://www.dartlang.org/community

Polymer's looks fairly recent:
https://github.com/Polymer/polymer/wiki/Code-of-Conduct

Angular's:
https://github.com/angular/code-of-conduct/blob/master/CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by kwalrath**
Fixes #539

Staged at:
https://kw-www-dartlang-1.firebaseapp.com/code-of-conduct
https://kw-www-dartlang-1.firebaseapp.com/community

### CODE REVIEW

File: src/community/index.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ description: "Communities, mailing lists, and bug databases for the Dart project
 
 Track the Dart project, get help, and talk with other Dart developers.
 
+We pledge to maintain an open and welcoming environment.
+For details, see our [code of conduct](/code-of-conduct).
+
+
```
Comment (kevmoo) [inline src/community/index.md]: extra line?

File: src/community/index.md [hunk 2]
```diff
@@ -21,6 +25,7 @@ Track the Dart project, get help, and talk with other Dart developers.
 * [+dartlang](https://plus.google.com/+dartlang) -
   The official Google+ account.
 
+
```
Comment (kevmoo) [inline src/community/index.md]: extra line?

File: src/community/index.md (hunk 1 above)
Comment (kwalrath) [reply src/community/index.md]: We often have two blank lines above headings, just to make it easier to find them. Is that a problem?

Comment (kevmoo) [reply src/community/index.md]: Not if it's intentional 😄

**Comment by kwalrath** [comment]
Devrel likes it, so I'm posting this.

**Comment by kwalrath** [comment]
It's up! https://www.dartlang.org/code-of-conduct

**Pull request merged**